.github/workflows/build_package.yml

```diff
@@ -0,0 +1,94 @@
+name: Build and Deploy
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build_package:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Install tree CLI tool
+      run: sudo apt-get install -y tree
+
+    - name: Display directory tree before building
+      run: tree
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.11'
+
+    - name: Set up Conda
+      uses: conda-incubator/setup-miniconda@v2
+      with:
+        environment-file: env.yml
+        activate-environment: mle-dev
+
+    - name: Install Poetry
+      run: |
+        curl -sSL https://install.python-poetry.org | python3 -
+        export PATH="$HOME/.poetry/bin:$PATH"
+
+    - name: Install dependencies
+      run: poetry install
+
+    - name: Build the package
+      run: poetry build
+
```

Comment:
Use `setuptools` as _build backend_ (instead of `poetry`).
So instead of all these lines have this..
```
    - name: Building the package
      run: |
        pip install --upgrade setuptools
        pip install --upgrade build
        python -m build
```

Reply:
Change workflow accordingly